core/src/main/resources/hudson/logging/LogRecorderManager/feeds.jelly

```diff
@@ -28,22 +28,22 @@ THE SOFTWARE.
     <div align="right">
         <a href="rss" class="yui-button link-button">
             <span class="leading-icon">
-                <l:svgIcon class="icon-small" tooltip="Atom feed"
-                           href="${imagesURL}/material-icons/svg-sprite-communication-symbol.svg#ic_rss_feed_24px"/>
+                <l:svgIcon class="icon-small" tooltip="Atom feed" viewBox="viewbox (default 0 0 16 16)"
```

Comment:
Is there a way to avoid providing a fixed view box every time?

Reply:
I think there should be an alternative to the viewbox changes here. Maybe a `transform: scale`